src/requirements.txt

```diff
@@ -0,0 +1,6 @@
+python-dateutil ~=2.8
+pandas ~=1.3
+ndjson ~=0.3
+pytz >=2021.1
+us ~=2.0
+shapely ~=1.6
```

Comment:
Based on what was generally current in July.

Reply:
I have shapely 1.7, but otherwise looks good.

Reply:
Sweet, will change that and merge. Thanks! 👍